Desktop/components/JASP/Widgets/FileMenu/DataLibrary.qml

```diff
@@ -70,6 +70,33 @@ Item
 		width:					rect.width
 		orientation:			Qt.Horizontal
 	}
+	
+	ScrollMoreIndicator
+	{
+		anchors
+		{
+			top:			secondseparator.bottom
+			topMargin:		-secondseparator.height / 2
+			left:			parent.left
+			right:			parent.right
```

Comment:
Should also set leftMargin? It seems like it should be anchored to the left side of the scrollbar, as it is done in the button? But it's not a big deal. I'm not really a stickler for this idea;-)

![image](https://github.com/user-attachments/assets/777cc0ed-7f78-4afd-b004-07ef7ced8d97)

Reply:
Yeah, it wouldve been prettier, but there are some other issues there as well.
Like, it wont scroll all the way down for some reason.

But I was a bit distracted by the labeleditor :p

Ill have a look later